content/2_Bronze/Intro_Sorting.mdx

```diff
@@ -173,6 +173,38 @@ Two ways to avoid this:
 </LanguageSection>
 
 ### Dynamic Arrays
+<PySection>
+
+
+
+```py
+import sys
+
+n = int(input())
+a = [int(i) for i in input().strip().split(' ')]
+numSwaps = 0
+
+for i in range(n):
+    currentSwaps = 0
+
+    for j in range(0, n - 1):
+        if a[j] > a[j + 1]:
+            a[j], a[j + 1] = a[j + 1], a[j]
+            currentSwaps += 1
+            
+    numSwaps += currentSwaps
+            
+    if currentSwaps == 0:
+        break
+        
```

Comment:
please make consistent with c++ and java sols

Reply:
In Python, we usually just use the `list.sort` method or the `sorted` function, which is both more readable and more efficient than implementing it manually.